dex/elrond_dex_farm/src/lib.rs

```diff
@@ -219,29 +233,35 @@ pub trait Farm:
         }
 
         let farming_token_id = self.farming_token_id().get();
-        let mut farming_token_amount = amount.clone();
+        let mut initial_farming_token_amount = &(&farm_attributes.initial_farming_amount * &amount)
```

Comment:
Indeed it's the same calculus. The reason I did not replace everywhere was to minimize the number of changed lines. But at a second thought it might improve readability. Will update

Reply:
done

Reply:
Resolved